disperser/common/chunkstore/chunk_writer.go

```diff
@@ -0,0 +1,103 @@
+package chunkstore
```

Comment:
I feel like chunkstore belongs outside `disperser`. Should it be under `relay`?

Reply:
Moved it to the `relay`.

I was actually a little confused about where this should live. The relay reads from it, but the encoder will be writing to it. But I guess `relay` is as good as anywhere (unless we decide to make it a top level directory, which I don't think it deserves to be).